ide/docker.api/src/org/netbeans/modules/docker/api/DockerAction.java

```diff
@@ -150,8 +150,7 @@ public List<DockerImage> getImages() {
                 String id = (String) json.get("Id");
                 long created = (long) json.get("Created");
                 long size = (long) json.get("Size");
-                long virtualSize = (long) json.get("VirtualSize");
-                ret.add(new DockerImage(instance, repoTags, id, created, size, virtualSize));
+                ret.add(new DockerImage(instance, repoTags, id, created, size));
```

Comment:
I suppose this is the reason why it doesn't load. If the field isn't there, `null` would be unboxed to `long` and cause an exception.

There is a `getOrDefault` method we could use for `Created` and `Size`, even though those two are unlikely to disappear I think.

Reply:
> There is a `getOrDefault` method we could use for `Created` and `Size`, even though those two are unlikely to disappear I think.

Done ✅. Is my first time.